### PR TITLE
Tests/graphql data blocks

### PR DESCRIPTION
**Pull request by JeremiGendron**
## Proposed changes

Load blocks into local object and start building it for expected data returned, following the recursive logic of graphql since every block refers to transactions which refer to wallets which refer to blocks.
## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (improve a current implementation without adding a new feature or fixing a bug)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build (changes that affect the build system)
- [ ] Docs (documentation only changes)
- [x] Test (adding missing tests or fixing existing tests)
- [ ] Other... Please describe:

## Checklist

- [ x I have read the [CONTRIBUTING](https://docs.ark.io/developers/guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: packages/core-graphql/__tests__/graphql.test.js [hunk 1]
```diff
@@ -21,63 +31,32 @@ describe('GraphQL', () => {
...
-      expect(testQueries).toBeObject()
+      expect(testData).toBeObject()
     })
 
-    describe('test query for Block', () => {
-      it('should have queries', () => {
-        expect(testQueries.block.length).toBeTruthy()
+    describe('loading blocks', () => {
+      references.data.blocks = testData.blocks
+      expect(references.data.blocks).toBe(testData.blocks)
```
Comment (air1one) [inline packages/core-graphql/__tests__/graphql.test.js]: What does this achieve ? 
Also, any `expect` should be inside some `it` block, I don't really see what it is doing here.

File: packages/core-graphql/__tests__/graphql.test.js [hunk 2]
```diff
@@ -21,63 +31,32 @@ describe('GraphQL', () => {
...
 
-        testQueries.block.forEach((block) => {
-        })
-      })
-    })
-
-    describe('test query for Blocks', () => {
-      it('should have queries', () => {
-        expect(testQueries.blocks.length).toBeTruthy()
+      it('should load transactions', () => {
```
Comment (air1one) [inline packages/core-graphql/__tests__/graphql.test.js]: What are we testing in this `it` block ? I don't see any `expect`.

**Comment by JeremiGendron** [comment]
No problem, will work on GQL during the next month!

**Comment by faustbrian** [comment]
Will close this so @air1one can write some initial tests that will serve as examples how things should be done.

**Pull request closed without merging**